Fastlane/Fastfile

```diff
@@ -0,0 +1,44 @@
+require_relative 'Configuration'
+
+default_platform :ios
+
+platform :ios do
+ 
+  desc "Runs all the tests"
+  lane :unit do
+    unit_test
+  end
+
+  desc "Builds alpha configuration"
+  lane :alpha do
+    build
+  end
+
+  private_lane :unit_test do
+    scan(
+      scheme: configuration.app_scheme,
+      devices: configuration.scan_devices,
+      clean: false,
+      prelaunch_simulator: true,
+      derived_data_path: './build_for_unit_testing'
+      )
+  end
+
+  private_lane :build do |options|
+    gym(
+      scheme: configuration.app_scheme,
+      output_directory: configuration.app_output_directory,
+      clean: options[:clean].nil? ? true : options[:clean],
```

Comment:
We don't need `.nil?` here for the ternary operator as Ruby will return false if options[:clean] is `nil`, also could probs simplify (if options[:clean] is anything but `nil` or `false` then return options[:clean], else return `true`)
```
options[:clean] || true
```